app/containers/Toast.js

```diff
@@ -20,15 +19,17 @@ const styles = StyleSheet.create({
 	}
 });
 
-export const LISTENER = 'Toast';
-
 class Toast extends React.Component {
 	static propTypes = {
 		theme: PropTypes.string
 	}
 
+	static mounted = null;
+
+	static show = (...args) => Toast.mounted && Toast.mounted.showToast(...args);
```

Comment:
It's really odd how `showToast` is a child of `mounted`.

Reply:
I do agree that it's odd, yeah